### ISSUE TITLE
Clean up comments in Collaboration Cycle operational guidance

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#47825]
## User Story
As a Governance team member, I want to be able to follow directions without navigating comments so that I can complete my task quickly and efficiently.

Assignee: @allison0034 
Peer Reviewer: @CherylEvans 

## Description
There are many comments piling up in the Collaboration Cycle operational docs in Confluence. The purpose of this ticket is to read through them all, identify if there is still an issue, fix it or escalate it (if it cannot be fixed easily) and resolve the comment.

## Impacted Artifacts
All operational docs nested under [Platform products and services/Collaboration Cycle](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1748860945/Collaboration+Cycle)

## Tasks
- [x] Complete a thorough review of every page in the Collaboration Cycle operational docs
- [x] When a comment is found, identify if the issue can be immediately resolved
   - [x] If yes, address the concern of the comment then mark the comment as resolved
   - [x] If no, document below the artifact and the issue in need of escalation (10/11/22 added new tickets to backlog: #48020, #48023, #48039)

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] All comments in Collaboration Cycle operational documentation have been resolved or escalated

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [ ] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
@CherylEvans FYI I added you as peer reviewer for this ticket. Please let me know if you have any questions.

**Comment by allison0034** [comment]
@shira if the comments are small do you want me to go ahead and make the change on the page and if they are larger do you want me to create a ticket for future work?

**Comment by shiragoodman** [comment]
@allison0034 yes. If you understand the comment, and can make the change, then go for it - make the change. If they're larger and it would require more thought, then create a ticket. Be sure to link all tickets you create within this ticket.

**Comment by allison0034** [comment]
@CherylEvans this is ready for your review. I have resolved all comments on the operational docs and created 3 new tickets.

**Comment by CherylEvans** [comment]
@Allison I compared the newly created tickets (Impacted artifacts) with their corresponding confluence document, and reviewed all the operational docs that were impacted.  It looks like all comments have been resolved.  Great job on such a comprehensive ticket!!  
I do have one question with regard to [How to complete a Design Intent review: Design](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2104426655/How+to+complete+a+Design+Intent+review+Design) that is nested under How to provide feedback for a Design Intent, should this page be deleted, or is something going to go there eventually?
